setup.cfg

```diff
@@ -34,14 +34,18 @@ zip_safe = False
 install_requires =
     six>=1.10
     isbnlib>=3.10.8
+    importlib-metadata>=6.11.0
 
 [options.extras_require]
 tests =
     pytest-black-ng>=0.4.0
     pytest-cache>=1.0
     pytest-runner>=2.6.2
-    pytest-invenio>=1.4.0
```

Comment:
We would need to implement locally in the module the mocking of entrypoints so that the tests will pass